__tests__/app.test.js

```diff
@@ -1,46 +1,83 @@
+/** * @jest-environment node */
 const request = require('supertest');
-const app = require('../src/app');
 const nock = require('nock');
+const app = require('../src/app');
+const { mockListOfJokes, mockRandomJokes, mockPersonalJokes } = require('../data/test-data');
 
-describe('My jokes API', () => {
-  describe('API GET /', () => {
-    it('should respond with a welcome message', done => {
-      request(app)
-        .get('/')
-        .then(res => {
-          expect(res.statusCode).toEqual(200);
-          expect(res.body.message).toEqual('Welcome to my jokes API!');
-          done();
-        });
-    });
-  });
-  it('that this is the jokes endpoint', done => {
+describe('GET / - Homepage', () => {
+  it('should respond with some homepage markup', async () => {
     request(app)
-      .get('/jokes')
+      .get('/')
       .then(res => {
```

Comment:
Because you are using a .then chained to a promise, this is why you don't need the async above. Async/await come in pairs, but they are useless without each other.